tests/enableCache/perspectives.js

```diff
@@ -76,17 +78,54 @@ describe(`tests/enableCache/perspectives.js, api: GET ${path} >`, () => {
   });
 
   it('basic get, response present in cache', (done) => {
-    redisCache.get('{\"where\":{}}', (cacheErr, reply) => {
-      if (reply) {
-        const jsonReply = JSON.parse(reply);
-        expect(jsonReply).to.have.length(ONE);
-        expect(jsonReply[ZERO].name).to.be.equal('___testPersp');
-        expect(jsonReply).to.have.deep.property('[0].lensId', lensId);
+    api.get(path)
+    .set('Authorization', token)
+    .expect(constants.httpStatus.OK)
+    .end((err, res) => {
+      if (err) {
+        done(err);
+      }
 
-        done();
-      } else {
-        throw new Error('Expected response value in cache');
+      redisCache.get('/v1/perspectives', (cacheErr, reply) => {
+        if (reply) {
+          const jsonReply = JSON.parse(reply);
+          expect(jsonReply).to.have.length(ONE);
+          expect(jsonReply[ZERO].name).to.be.equal('___testPersp');
+          expect(jsonReply).to.have.deep.property('[0].lensId', lensId);
+
+          done();
+        } else {
+          console.log(cacheErr);
+          throw new Error('Expected response value in cache');
+        }
+      });
+    });
+  });
+
+  it('basic get, response present in cache with enableApiActivityLogs',
+    (done) => {
+    tu.toggleOverride('enableApiActivityLogs', true);
```

Comment:
toggleOverride here without saving previous feature enablement value and restoring it after